start/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Get Started"
+---
+
+We are a community of civic hackers, data wranglers and ordinary citizens intrigued and excited by the possibilities of combining technology and information for good – making government more accountable, culture more accessible and science more efficient.
+
+We focus on making things - whether that’s apps, insights or tools. We have a strong preference for [open data](http://opendefinition.org/) and [free/open source software](https://opensource.org/).
+
+We are part of [Open Knowledge International](https://okfn.org/) and operate as a collaborative community which anyone can join. Labs is organized through a public [GitHub Repository](https://github.com/okfn/okfn.github.com).
+
+## Types of Activities
+
+### Data Wrangling
+
+Data wrangling can be described as the process of getting, cleaning and using data. The following list covers almost all activities we do.
+
+LIST
+
+Clicking [here]({{site.baseurl}}/data/tutorial), you can find a walk-through of the process of getting, cleaning and publishing data online. 
+#### Skills required
```

Comment:
should have a blank line above. I will let you fix these items in a separate PR.